themes/classic/_dev/js/components/product-miniature.js

```diff
@@ -25,28 +25,35 @@
 import $ from 'jquery';
 
 export default class ProductMinitature {
-  init(){
+  init() {
     $('.js-product-miniature').each((index, element) => {
       const FLAG_MARGIN = 10;
-      let $discount = $(element).find('.discount-product');
-      let $onsale =  $(element).find('.on-sale');
-      let $new = $(element).find('.new');
-      if($discount.length){
-        $new.css('top', $discount.height() * 2 + FLAG_MARGIN);
-        $discount.css('top',-$(element).find('.thumbnail-container').height() + $(element).find('.product-description').height() + FLAG_MARGIN);
+      let discountElems = $(element).find('.discount-product');
```

Comment:
You can use const for `discountElems`, `onSaleElems`,  and `newElems` You never change the value.

Reply:
good point. added in https://github.com/PrestaShop/PrestaShop/pull/10267/commits/03d0b59554c59bd10ed5073ccbb173d1ea011959. thx